src/services/iot/index.ts

```diff
@@ -159,3 +167,5 @@ export const producedEventsSupportedServices = [
 export const producedDeployOutputTypes = [];
 
 export const consumedDeployOutputTypes = [];
+
+export const supportsTagging = true;
```

Comment:
The Handel docs also don't show that Tags are supported, but we apparently do add tags to the stack. Can we update those docs as part of this PR?